packages/graphql-server/nodemon.json

```diff
@@ -2,5 +2,5 @@
   "watch": ["src"],
   "ext": "ts,ejs,graphql",
   "ignore": ["src/**/*.test.ts", "src/schema.graphql"],
-  "exec": "yarn build:schema && ts-node -r tsconfig-paths/register -P tsconfig.json ./src/app.ts"
+  "exec": "ts-node -r tsconfig-paths/register -P tsconfig.json ./src/app.ts"
```

Comment:
If you remove `yarn build:schema` here, can you change the `start:dev` to 
`cross-env REAPIT_ENV=LOCAL NODE_ENV=development yarn build:schema && nodemon`
in `package.json`, so it only builds one time, before start the dev server